.github/workflows/cd.yml

```diff
@@ -120,15 +120,13 @@ jobs:
           onepassword-api-key: ${{ secrets.OP_CONNECT_TOKEN }}
           app-name: ${{ github.event.repository.name }}
 
-  # TODO: Run integration test suite here
-
-  check-staging-health:
```

Comment:
we should keep this step but edit it to fetch staging health and check for version to make sure we don't run tests till the version matches the new one else we could be running tests on an old container

Reply:
CD only deploys via Terraform. I fixed the Terraform to [`wait_for_completion`](https://github.com/WalletConnect/echo-server/blob/main/terraform/ecs/main.tf#L112) so now we know that by the time the deploy step is done the latest version is deployed.

Reply:
🤝